app/components/TableSearch.tsx

```diff
@@ -50,10 +50,9 @@ export default function TableSearch({
               return newQueryConfig;
             });
             setQueryConfig(newQueryConfig);
-          } else {
-            searchSettings.searchString = e.target.value;
-            setSearchSettings({ ...searchSettings });
           }
+          searchSettings.searchString = e.target.value;
+          setSearchSettings({ ...searchSettings });
```

Comment:
@mateoclarke the fix was up here. the bug was that we were skipping the update to `searchSettings` when the input was cleared. 

btw, `searchSettings` is holding the local state of the search string and search field, and clicking the **Search** button or clearing the input completely is what triggers the `queryConfig` state to be updated and fire off a new Hasura API call.